vignettes/unusual_tables.Rmd

```diff
@@ -0,0 +1,85 @@
+---
+title: "Unusual Tables"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Unusual Tables}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(tlang)
+library(tibble)
+library(dplyr)
+library(tidyr)
+```
+
+
+# Multiple columns of stub
+
+It is not all that unusual for listing (and some tables) to have multiple stub columns. When this happens it is often easier to not use thee stub functions for gt. An example of a table like this is the "Summary of Number of Subjects by Site" from the CDISC pilot.
+![Image of the Summary of Number of Subjects By Site table from the CDISC pilot](images/summary_subj_by_site.png)
+
+To make this table the values will be long with "Pooled Id" and "Site Id" as separate columns like normal. We also will still need a column for the parameters even though they are all the same. 
+
+```{r}
+data <- tribble(
+  ~`Pooled Id`,  ~`Site Id`,
+"701",  "701",
+"703",  "703",
+"704",  "704",
+"705",  "705",
+"708",  "708",
+"709",  "709",
+"710",  "710",
+"713",  "713",
+"716",  "716",
+"718",  "718",
+"900",  "702",
+"900",  "706",
+"900",  "707",
+"900",  "711",
+"900",  "714",
+"900",  "715",
+"900",  "717",
+"Total", " ") %>%
+  crossing(col1 = c("Placebo (N=86)",
+                   "Xanomeline Low Dose (N=84)",
+                   "Xanomeline High Dose (N=84)",
+                   "Total (N=254)"),
+           col2 = c("ITT", "Eff", "Com")) %>%
+  mutate(val = rpois(216, 15), # Here I am just faking the data for display purposes 
+         param = "val")
+
+```
+
+Once we have the data in the standard ARD format we can make the tfrmt. What makes this tfrmt different is we won't include group or label. This will make it so the table will just be ordered based on the `col_plan()`. Because the `col_plan()` follows the conventions of `select()` we are going to start with the lowest level columns, and then select the order of the spanninig columns. 
```

Comment:
```suggestion
Once we have the data in the standard ARD format we can make the tfrmt. What makes this tfrmt different is we won't include group or label. This will make it so the table will just be ordered based on the `col_plan()`. Because the `col_plan()` follows the conventions of `select()` we are going to start with the lowest level columns, and then select the order of the spanning columns. 
```